ePayments API documentation (#92)
vippsas/vipps-epayment-api: features/long-living-payments.md

<!-- START_METADATA
---

> title: Long living payments

Reviewer: ```suggestion
title: Long-living payments
```

hide_table_of_contents: true
pagination_next: null
pagination_prev: APIs/epayment-api/getting-started
---
END_METADATA -->

> 

Reviewer: ```suggestion

# Long-living payments

```

The ePayment API supports long-living payments for paymentMethod type as `WALLET`, where the merchant can specify the expiration time when initiating the payment with POST:/epayment/v1/payments.

The expiration time is set with the expiresAt parameter.

> See [Long living payments](https://vippsas.github.io/vipps-developer-docs/docs/vipps-solutions/long-expiry-time-for-payments-to-merchants) for more details.

Reviewer: ```suggestion
See [Extended expiration for payments to merchants](https://vippsas.github.io/vipps-developer-docs/docs/vipps-solutions/long-expiry-time-for-payments-to-merchants) for more details.
```

Reviewer: I think we should actually move the details they need from that solutions section and add links from the Solution section to here, for more details.
